packages/driver/cypress/integration/util/breaking_change_warnings_spec.js

```diff
@@ -0,0 +1,41 @@
+describe('blob-util 2.x', () => {
+  it('arrayBufferToBlob', () => {
+    cy.on('fail', (err) => {
+      expect(err.message).to.include('no longer returns a `Promise`')
+    })
+
+    Cypress.Blob.arrayBufferToBlob('1234').then((blob) => {
+      // it should fail.
+    })
+  })
```

Comment:
A `(done)` callback parameter must be yielded to guarantee that the `cy.on('fail', ...)` hits accordingly otherwise the test can pass if the error handle is not invoked.

```js
it('arrayBufferToBlob', (done) => {
    cy.on('fail', (err) => {
      expect(err.message).to.include('no longer returns a `Promise`')
	  done()
    })

    Cypress.Blob.arrayBufferToBlob('1234').then((blob) => {
      // it should fail.
    })
  })
```